core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp

```diff
@@ -128,10 +143,19 @@
             "ranger.jpa.jdbc.driver": "org.postgresql.Driver",
             "ranger.jpa.jdbc.url": "jdbc:postgresql://{{ RANGER_DB_HOST }}/{{ RANGER_DB_NAME }}",
             "ranger.audit.source.type": "solr",
-            "ranger.authentication.method": "ACTIVE_DIRECTORY",
-            "ranger.ldap.ad.domain": "{{ LDAP_DOMAIN }}",
-            "ranger.ldap.ad.base.dn": "{{ LDAP_SYNC_SEARCH_BASE }}",
-            "ranger.ldap.ad.user.searchfilter": "({{ LDAP_NAME_ATTRIBUTE }}={0})"
+            "ranger.authentication.method": "{{ LDAP_METHOD }}",
+            "ranger.ldap.ad.url": "{{ LDAP_URL }}",
+            "ranger.ldap.ad.base.dn": "{{ LDAP_USER_SEARCH_BASE }}",
+            "ranger.ldap.ad.user.searchfilter": "{{ LDAP_USER_SEARCH_FILTER }}",
+            "ranger.ldap.ad.bind.dn": "{{ LDAP_BIND_DN }}",
+            "ranger.ldap.ad.bind.password": "{{ LDAP_BIND_PASSWORD }}",
+            "ranger.ldap.url": "{{ LDAP_URL }}",
+            "ranger.ldap.base.dn": "{{ LDAP_USER_SEARCH_BASE }}",
+            "ranger.ldap.bind.dn": "{{ LDAP_BIND_DN }}",
+            "ranger.ldap.bind.password": "{{ LDAP_BIND_PASSWORD }}",
+            "ranger.ldap.group.searchbase": "{{ LDAP_GROUP_SEARCH_BASE }}",
+            "ranger.ldap.user.searchfilter": "{{ LDAP_USER_SEARCH_FILTER }}",
```

Comment:
We don't need search filter, we can use: "ranger.ldap.ad.user.searchfilter": "({{ LDAP_NAME_ATTRIBUTE }}={0})"